doc/user/content/serve-results/sink/s3_compatible.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "S3 Compatible Object Storage"
+description: "How to export results from Materialize to S3 compatible object storage"
```

Comment:
Oh ... we should update
https://preview.materialize.com/materialize/33752/sql/copy-to/#copy-to-s3 to specify s3 and s3-compatible
as well as remove the preview?